lib/response.ts

```diff
@@ -159,6 +159,19 @@ function parseXmlAndVersion(rawAssertion, cb) {
 
     xml = xmlBeautify(xml);
 
+    const statusValue =
+      xml.Response &&
+      xml.Response.Status &&
+      xml.Response.Status.StatusCode &&
+      xml.Response.Status.StatusCode['@'] &&
+      xml.Response.Status.StatusCode['@'].Value;
+    const statusParts = statusValue ? statusValue.split(':') : statusValue;
```

Comment:
@ukrocks007 

`const statusParts = statusValue ? statusValue.split(':') : statusValue;` Should we handle the case if the `statusValue` is `undefined`?

Reply:
I have handled it using the ternary operator.

```
const status = statusParts ? statusParts.length > 0 ? statusParts[statusParts.length - 1] :
 undefined: undefined;
```